gdcdictionary/schemas/gene_drug_association.yaml

```diff
@@ -0,0 +1,76 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "protein_drug_association"
```

Comment:
Try to be consistent in naming. Either rename file to be `protein_drug_association` or change class to be `gene_drug_association`